src/main/java/br/com/caelum/vraptor/util/jpa/extra/ParameterLoader.java

```diff
@@ -69,32 +73,29 @@ public ParameterLoader(EntityManager em, HttpServletRequest request, ParameterNa
 		this.flash = flash;
 	}
 
+	@Accepts
 	public boolean containsLoadAnnotation(ControllerMethod method) {
```

Comment:
> By the way, does this version of VRaptor has the `@Lazy` annotation? 
Then any dependencies would be injected unnecessarily.

CDI already lazy resolve it's dependencies. No need to this using vraptor 4